$OpenBSD: patch-plugins_video_H_264_h264-x264_cxx,v 1.1 2012/08/25 13:17:54 ajacoutot Exp $

LOCALBASE/include/libavutil/common.h:170: error: 'UINT64_C' was not declared in this scope,
so move headers so that __STDC_CONSTANT_MACROS is defined at the right time.

--- plugins/video/H.264/h264-x264.cxx.orig	Sat Aug 25 09:14:53 2012
+++ plugins/video/H.264/h264-x264.cxx	Sat Aug 25 09:15:26 2012
@@ -36,14 +36,14 @@
  * $Date: 2012/08/25 13:17:54 $
  */
 
+#include "../common/ffmpeg.h"
+#include "../common/dyna.h"
+
 #ifndef PLUGIN_CODEC_DLL_EXPORTS
 #include "plugin-config.h"
 #endif
 
 #include <codec/opalplugin.hpp>
-
-#include "../common/ffmpeg.h"
-#include "../common/dyna.h"
 
 #include "shared/h264frame.h"
 #include "shared/x264wrap.h"
